src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -590,7 +590,13 @@ enum SMART_ACTION
     SMART_ACTION_MUSIC                              = 216,    // SoundId, onlySelf, type
     SMART_ACTION_RANDOM_MUSIC                       = 217,    // SoundId1, SoundId2, SoundId3, SoundId4, onlySelf, type
 
-    SMART_ACTION_AC_END                             = 218,    // placeholder
+    SMART_ACTION_CUSTOM_CAST                        = 218,    // spellId, castflag, bp0, bp1, bp2
+    SMART_ACTION_CONE_SUMMON                        = 219,    // entry, duration (0 = perm), dist between rings, dist between earch summon in a row, length of cone, width of cone (angle)
+    SMART_ACTION_PLAYER_TALK                        = 220,    // creature_text.id, yell? (0/1)
```

Comment:
This is not right yet, its  trinity_strings  DB ... I originally wanted to do it with creature_text